ViAn/GUI/analysisslider.h

```diff
@@ -10,7 +10,7 @@ class AnalysisSlider : public QSlider {
 
 public:
     explicit AnalysisSlider(Qt::Orientation orientation, QWidget *parent = 0);
-    void add_slider_rect(double start, double end);
+    void add_slider_interval(int start_frame, int end_frame);
     void clear_rects();
```

Comment:
please change clear_rects to clear_slider or similar. This name makes sense internally but perhaps not to an outside caller